job-dsls/jobs/srcclr_scan_job.groovy

```diff
@@ -0,0 +1,79 @@
+def repoFile = readFileFromWorkspace("repository-list.txt")
+def repoList = repoFile.readLines()
+
+for (repo in repoList) {
+
+    def jobName = "${JOB_PATH}/srcclr-scan-${repo}"
+
+    job(jobName) {
+
+        description("Job responsible for SourceClear verification of ${repo}")
+
+        parameters {
+            choiceParam('SCAN_TYPE', ['scm', 'binary'])
+            stringParam('SRCCLR_INVOKER_REPO_URL','')
+            stringParam('URL','')
+            stringParam('VERSION', '')
+            stringParam('PACKAGE','')
+            stringParam('NAME', '')
+            stringParam('MVNPARAMS', '')
+            choiceParam('PROCESSOR_TYPE', ['cve', 'cvss'])
+            booleanParam('RECURSIVE', false)
+            booleanParam('DEBUGGING', false)
+            booleanParam('TRACING', false)
+            stringParam('SCMVERSION', '')
+            stringParam('THRESHOLD', '1','Threshold from 1 to 10 for cvss processor')
+        }
+
+
+        environmentVariables{
+            groovy('''
+          def map = [:]
+          map.put("RECURSE", Boolean.valueOf("${RECURSIVE}") ? "--recursive" : "")
+          map.put("DEBUG", Boolean.valueOf("${DEBUGGING}") ? "-d" : "");
+          map.put("TRACE", Boolean.valueOf("${TRACING}") ? "--trace" : "")
+          map.put("MVNPARAMETER", "${MVNPARAMS}" !="" ? "--maven-param=${MVNPARAMS}":"")
+          map.put("SCMVERSIONPARAM", "${SCAN_TYPE}" == "scm" ? " --ref=${SCMVERSION}":"")
+          return map
+        ''')
+        }
+
+
+        label("kie-rhel7")
```

Comment:
I would add `and !master` leaving the final label as `label("kie-rhel7 & !master")`

Reply:
+1

Reply:
so why don't you @akoufoudakis limit running off the master instance?